cmd/ethereum_rust/cli.rs

```diff
@@ -19,6 +19,7 @@ pub fn cli() -> Command {
                 .value_name("PORT")
                 .action(ArgAction::Set),
         )
+        .arg(arg!(-v --verbose "Max verbosity for logs").required(false))
```

Comment:
Sorry, I was thinking something more like what lighthouse has

```
      --logfile-debug-level <LEVEL>
          The verbosity level used when emitting logs to the log file. [default:
          debug] [possible values: info, debug, trace, warn, error, crit]
```

the flag name could be `--log-level`. Thoughts?

Reply:
Oh okay, changing that now to match more levels of logs 🫡